### PR TITLE
Offer list task1

### PR DESCRIPTION
**Pull request by DianaProkopovich**

### CODE REVIEW

**Review by Zairbekov — APPROVED** [review]

File: angular.json [hunk 1]
```diff
@@ -126,5 +126,8 @@
       }
     }
   },
-  "defaultProject": "exadel-app"
+  "defaultProject": "exadel-app",
+  "cli": {
+    "analytics": false
+  }
```
Comment (abarmina) [inline angular.json]: this is automatically added, we don't need it, can you delete it?

File: src/app/pages/home-page/offer-list/offer-item/offer-item.component.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                    <mat-icon aria-hidden="false" aria-label="people">star</mat-icon>
+                    <mat-icon aria-hidden="false" aria-label="people">star</mat-icon>
+                </div>
+                <mat-icon aria-hidden="false" aria-label="people">visibility</mat-icon>
+                <span class="offer-card-properties__viewsCount">{{offer.numberOfViews}}</span>
+                <mat-icon aria-hidden="false" aria-label="people">people</mat-icon>
+                <span class="offer-card-properties__usesCount">{{offer.numberOfUses}}</span>
+            </div>
+    </mat-card-header>
+</mat-card> -->
```
Comment (abarmina) [inline src/app/pages/home-page/offer-list/offer-item/offer-item.component.html]: will we need it later or can we delete it?

File: src/app/pages/home-page/offer-list/offer-item/offer-item.component.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      <mat-card-title class="offer-card-header__title">{{
+        offer.title
+      }}</mat-card-title>
+      <mat-card-subtitle class="offer-card-header__subtitle">{{
+        offer.vendorName
+      }}</mat-card-subtitle>
+    </mat-card-title-group>
+    <div class="offer-card__discount">{{ offer.discount }}</div>
+    <div class="properties-wrapper">
+      <div class="offer-card__properties offer-card-properties">
```
Comment (tlazarchuk) [inline src/app/pages/home-page/offer-list/offer-item/offer-item.component.html:19]: The same classes

File: src/app/shared/mock-offers.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
...J6LxaEZ40tQYVGi7YR5GBkm7q0sTJn7DTGSO74pwvrWfzCunQHocabHOGPN9Nu2NWfebdyGNtJu2KZu2Oi5rSdrB2Hyf9oOXxPzn6FGpOZsyq0M0bkUjBr6G6YybciA+JY6Fjm8Vie0dJOnZDh10P9Jycn56ukPwSu3pGkUHdBgz0KT0EW/jHDu2KFY2az809W9bGUkxoLvxtmfPKZbYBsthp+dSbKLnfSU61e6dFcGJytKjWZwGKQlKYPIDJtZ69Cue/wDFpNMKM2fYvV9dpVmzuEBqo+0SoGmn6WMDzb7ChOmTe/NvPnJ2eXtid9nA9UrUxG2UuAQKkl6egsrojAUtyaKyrCTFM/nW8KgTHJCcDWb6VBrJUzUYN2+hLU9Kf/4Vds5IJYKz5uNj64eyZQ4Hl3Mcveq7cGQFgDGhR/KdmoFpyMbiwFFqGl0SdNkWtkWjE0LaPUZbVfEV20o39sbwpBeaRommbrs1HGk6tPQZ4LQLnuWw/wz7SCO+eqGzwKqeg+seSBd7PQnj9rb9eVZp8frMwY+piTtaqRHAhkWVafmAyp/7ViZLP6sowlGlEsXR9TpqPtFSm2BEmcGrP6YB7kg7/UGO+ILOnwlfmrxBJr2f/tq68E3FvxoHfX3y07kT5PJRhQfnZu+fnIy7nZIvQltNjXFwlG5QOnSzs7S0s7kyG5lpmfFMTgpJZWlDTkocmSaJZyW9+OD11mbc1kV02IhJdX9t40d8V4zLtZeHcqlu4YALQe/mSxKBdKZbiw831P6tZG7qtWS9c55s5jUFoOF4cHLjF8Te1AaAk+snnhiWcDS1Hu9eUu7dxLTVoRQ64rjlwn+vyIe8z9mcNnP/W6+lbu+e5DIYEZ74plfRk02aXvLTdYWuO/+qcc7h7gxbdam13Nu8hy7Fp1Lt9P3FKU9ryR8bJYkZBnfmsEgkWr3rUsxZrq+E1I1yG50wfeDSwoHb4/Yj1ekAvSn/LB+95dJBU4M/Xm4u9Vv6f63fzJy8VUYOarHjYsl4dfDw6U/N2mFDhK7N3zulaKcufB/fHev4Jbf5je3qG9e47qXW9vY92+8Y8/zz//BDdu3Phi4f9BKgRBEARBEARBEARBEARBEARBEARBEARBEARBEARBEARBEARBEARBEARBEARBEARBEARBEAT5e/M/o07bh5eyxSQAAAAASUVORK5CYII=',
```
Comment (tlazarchuk) [inline src/app/shared/mock-offers.ts:9]: It doesn't look like photoUrl

File: angular.json (hunk 1 above)
Comment (DianaProkopovich) [reply angular.json]: deleted

File: src/app/shared/services/offer.service.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import { OFFERS } from '../mock-offers';
+import { Offer } from '../models/offer';
+import { Injectable } from '@angular/core';
```
Comment (tlazarchuk) [inline src/app/shared/services/offer.service.ts:3]: You should import first from @angular/core and after another imports

File: src/app/pages/home-page/offer-list/offer-item/offer-item.component.html [hunk 6]
```diff
@@ -0,0 +1,62 @@
+<mat-card class="offer-card">
+  <mat-card-header *ngIf="offer">
+    <img
+      class="offer-card__image"
+      mat-card-avatar
+      src="{{ offer.photoUrl }}"
+      alt=""
```
Comment (tlazarchuk) [inline src/app/pages/home-page/offer-list/offer-item/offer-item.component.html:7]: Why you use an empty alt?

**Pull request merged**